src/app/pages/sharing/components/shares-dashboard/iscsi-card/iscsi-card.component.ts

```diff
@@ -118,15 +113,18 @@ export class IscsiCardComponent implements OnInit {
       untilDestroyed(this),
     ).subscribe({
       next: () => {
-        this.getIscsiTargets();
+        this.dataProvider.load();
       },
       error: (err) => {
         this.dialogService.error(this.errorHandler.parseError(err));
       },
     });
   }
 
-  private getIscsiTargets(): void {
-    this.dataProvider.load();
+  setDefaultSort(): void {
+    this.dataProvider.setSorting({
+      active: 0, // index column
```

Comment:
Instead of a comment, it would be better to create a variable and use that instead of just `0`. Like `let activeColumnIndex = 0`